parsers/US_NY.py

```diff
@@ -8,13 +8,21 @@
 import pandas as pd
 
 mapping = {
-           'Dual Fuel': 'unknown',
+           'Dual Fuel': 'gas', # NYISO states "Power produced by facilities capable of using natural gas and/or other fuels"
+                # <5% is oil burn from 2016 State of NYISO Market Report
            'Natural Gas': 'gas',
            'Nuclear': 'nuclear',
-           'Other Fossil Fuels': 'unknown',
-           'Other Renewables': 'unknown',
-           'Wind': 'wind',
-           'Hydro': 'hydro'
+           'Other Fossil Fuels': 'coal', # NYISO states "Power produced by either coal or oil" - assume coal, oil only power plants only as last resort
+           'Other Renewables': 'biomass', # NYISO states "Power produced by either solar energy, refuse, methane or wood"
```

Comment:
I'm not sure I'd assume biomass here as solar might be a significant chunk.
Having 253MW during the night only gives us an idea of biomass production, but not of solar production.
We should look around noon, and if numbers are close to 250, then we can exclude solar, although this isn't future proof (when NYISO adds more solar capacity).

Reply:
Solar is not a significant chunk of anything in NY
Page 42 - http://www.nyiso.com/public/webdocs/media_room/publications_presentations/Power_Trends/Power_Trends/2016-power-trends-FINAL-070516.pdf

Solar is not coming in vast supplies to NY, and it is not produced during the winter.

Can you trust me on this one? - there is not a way to note & link to EM & pthyon files all the resources I am pulling this "well informed information" from.